### PR TITLE
Bug fix for select component

### PR DESCRIPTION
**Pull request by AjaySingh5**
### Please review these reminders and either check the box or explain why not:

- [x] Read and followed the [contributing guidelines](https://eds.ethoslabs.io/#/Guidelines?id=section-contribute)?
- [ ] Component is exported from [index.js](https://github.com/getethos/ethos-design-system/blob/master/src/components/index.js)?
- [ ] Type is exported from [index.d.ts](https://github.com/getethos/ethos-design-system/blob/master/src/components/index.d.ts) so Typescript users can consume it? Added a test and ran `yarn test:types`?
- [x] Bumped the yarn version?

### Referencing PR or Branch (e.g. in CMS or monorepo):

- reverts what happened in https://github.com/getethos/ethos-design-system/pull/436

### Screenshots and extra notes:

- Basically the validators for select didn't like that we used empty string for value [code](https://github.com/getethos/ethos/blob/d9e593a68944a7e94c9159dc1e295fee947825ec/frontend/src/main/InsuranceApp/PreInterview/Questions/validators.ts#L110)
- This fixes that while still fixing the underlying issue of the console error being thrown on initial select Blur

### CODE REVIEW

File: src/components/Select/Select.js [hunk 1]
```diff
@@ -80,10 +77,10 @@ export const Select = ({
     let resolvedValues = ''
     // react-select multi select case
     if (Array.isArray(userSelection)) {
-      resolvedValues = userSelection.map((selection) => selection.value)
+      resolvedValues = userSelection?.map((selection) => selection?.value)
```
Comment (WesleyDRobinson) [inline src/components/Select/Select.js:80]: one last check, do you need to add a `.filter(el => el)` to strip any `undefined` from resolvedValues?

Comment (AjaySingh5) [reply src/components/Select/Select.js]: Absolutely, [90097bb](https://github.com/getethos/ethos-design-system/pull/437/commits/90097bb5dcaf56e3e7983ec3bdefd8cd4eeb6573)

Comment (AjaySingh5) [reply src/components/Select/Select.js]: Per slack DMs, [d29d345](https://github.com/getethos/ethos-design-system/pull/437/commits/d29d345adbb44c8e803aad270b867f9dc2094784)

**Review by WesleyDRobinson — APPROVED** [review]
